python/pyspark/sql/connect/column.py

```diff
@@ -389,3 +389,46 @@ def __nonzero__(self) -> None:
 
 
 Column.__doc__ = PySparkColumn.__doc__
+
+
+def _test() -> None:
+    import os
+    import sys
+    import doctest
+    from pyspark.sql import SparkSession as PySparkSession
+    from pyspark.testing.connectutils import should_test_connect, connect_requirement_message
+
+    os.chdir(os.environ["SPARK_HOME"])
+
+    if should_test_connect:
+        import pyspark.sql.connect.column
+
+        globs = pyspark.sql.column.__dict__.copy()
```

Comment:
```suggestion
        globs = pyspark.sql.connect.column.__dict__.copy()
```

Oops, it was my misake from another PR.